packages/labs/react/src/create-component.ts

```diff
@@ -84,16 +84,34 @@ const setRef = (ref: React.Ref<unknown>, value: Element | null) => {
   }
 };
 
-type Events<S> = {
-  [P in keyof S]?: (e: Event) => unknown;
-};
-
 type StringValued<T> = {
   [P in keyof T]: string;
 };
 
 type Constructor<T> = {new (): T};
 
+type EventString<T = unknown> = string & {
```

Comment:
Specify the type parameter constraint and default to `Event`

```suggestion
type EventString<T extends Event = Event> = string & {
```

Reply:
Done :)